Automattic/vip-go-mu-plugins
# PIE-3814 Put Jetpack in offline mode when we're launching a site in the network

## Description
This PR adds some changes to put Jetpack in offline mode when we're going through the launch flow.

To do so we're using the `launching` flag in the `vip_launch_tools` cache group.

When that is set to true, we'll put Jetpack in offline mode. This is to avoid sending wrong data to Jetpack while we're doing the Search&Replace.

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test

1. Check out PR.
1. On a multisite, with Jetpack active, select a network site and run ` wp cache set launching true vip_launch_tools 0 --url=<networkSiteURL>`
1. For that site you should see Jetpack go into offline mode.
![image](https://github.com/Automattic/vip-go-mu-plugins/assets/668251/69007b98-5b1e-4f92-93f9-e008028a164e)
1. Other sites in the network should not be affected.
1. Remove the cache with `wp cache delete launching vip_launch_tools --url=<networkSiteURL>`. Jetpack should be active again on that specific site.
